frontend/src/scenes/trends/persons-modal/personsModalLogic.ts

```diff
@@ -142,6 +142,25 @@ export const personsModalLogic = kea<personsModalLogicType>([
                 return aggregationLabel(isGroupType(firstResult) ? firstResult.group_type_index : undefined)
             },
         ],
+        propertiesTimelineFilterFromUrl: [
```

Comment:
This could benefit from a simple unit test.

Reply:
Added a logic test